lib/config.go

```diff
@@ -514,6 +515,11 @@ version 1.14, this option will no longer exist.
 			logrus.Debugf("found valid runtime '%s' for runtime_path '%s'\n",
 				runtime, handler.RuntimePath)
 		}
+
+		// Validate the system registries configuration
+		if _, err := sysregistriesv2.GetRegistries(nil); err != nil {
+			return fmt.Errorf("invalid /etc/containers/registries.conf: %q", err)
+		}
```

Comment:
The code looks fine to me, my only question is if it is valid to pass 'nil' to GetRegistries here?  I know we do so for the tests, but thought we'd need a systemcontext for non tests.  @vrothberg WDYT?

Reply:
that should be totally fine, we're passing `nil` pretty much anywhere else afaict (see #2028)